pytorch_lightning/core/lightning.py

```diff
@@ -111,6 +113,8 @@ def __init__(self, *args, **kwargs):
         self._running_manual_backward = False
         self._current_hook_fx_name = None
         self._current_dataloader_idx = None
+        self._invalid_loss_strategy = InvalidLossStrategy.SKIP_IF_ANY
```

Comment:
Why did you change it to be a protected attribute (instead of just the property)